src/main/java/ai/promoted/delivery/client/PromotedDeliveryClient.java

```diff
@@ -383,10 +384,10 @@ private void fillInRequestFields(Request request) {
    */
   private void ensureClientTimestamp(Request request) {
     if (request.getTiming() == null) {
-      request.setTiming(new Timing());
+      request.setTiming(Timing.newBuilder().build());
```

Comment:
Ditto for this method and if Request is a Proto or POJO.

Reply:
Ack